app/assets/stylesheets/global.scss

```diff
@@ -10,6 +10,20 @@ h1, h2, h3, h4, h5, h6 {
   font-family: $headline-fonts;
 }
 
+h1 {
+  font-weight: 300;
+}
+
+h2 {
+  color: #4a4a4a;
+  font-weight: 700;
+}
+
+h3 {
+  color: #4a4a4a;
```

Comment:
Instead of repeating the color styles for `h2` and `h3`, can you do the following?

```css
h2, h3 {
  color: $text-primary;
}
```